src/js/main.js

```diff
@@ -183,4 +184,92 @@ Site.Menu = {
   }
 };
 
+Site.Countdown = {
+  $countdown: $('#countdown'),
+  $countdownDays: $('#countdown-days'),
+  $countdownHours: $('#countdown-hours'),
+  $countdownMinutes: $('#countdown-minutes'),
+  $countdownSeconds: $('#countdown-seconds'),
```

Comment:
As a convention we do these inside `init()` because theres cases where the object need to reinit and have new values on element selectors.